qa-ibd.planx-pla.net/portal/gitops.json

```diff
@@ -60,6 +60,13 @@
           "body": "The Workspace provides a secure cloud environment and features Jupyter Notebooks and RStudio",
           "link": "/workspace",
           "label": "Run analysis"
+        }, 
+        {
+          "name": "Study Viewer",
+          "icon": "query",
+          "body": "<Study Viewer Description>",
+          "link": "/study-viewer/study_viewer_file",
+          "label": "Study Viewer"
         }
```

Comment:
we can remove this; we don't need to add a card to the homepage, just a button in the navigation bar